client/css/create.css

```diff
@@ -1,28 +1,36 @@
 .view-create .box {
-  margin-top: 15px; }
-  .view-create .box > :last-child {
-    margin-bottom: 0px; }
-  .view-create .box.right {
-    background-color: #8596ab;
-    color: white;
-    padding: 15px; }
-    .view-create .box.right h1, .view-create .box.right h2, .view-create .box.right h3, .view-create .box.right h4, .view-create .box.right h5, .view-create .box.right h6 {
-      color: white;
-      font-family: "News Cycle", sans-serif;
-      font-weight: normal;
-      margin: 0 0 15px 0; }
-    .view-create .box.right .button {
-      width: 100%;
-      background: #526378;
-      border: 1px solid #29313b; }
-      .view-create .box.right .button:hover {
-        background: #3e4a59; }
-  .view-create .box.left {
-    background-color: white;
-    padding: 15px; }
-    .view-create .box.left h1, .view-create .box.left h2, .view-create .box.left h3, .view-create .box.left h4, .view-create .box.left h5, .view-create .box.left h6 {
-      color: #810814;
-      margin: 0 0 15px 0; }
+  margin-top: 15px;
+}
+.view-create .box > :last-child {
+  margin-bottom: 0px; 
+}
+.view-create .box.right {
+  background-color: #8596ab;
+  color: white;
+  padding: 15px; 
+}
+.view-create .box.right h1, .view-create .box.right h2, .view-create .box.right h3, .view-create .box.right h4, .view-create .box.right h5, .view-create .box.right h6 {
+  color: white;
+  font-family: "News Cycle", sans-serif;
+  font-weight: normal;
+  margin: 0 0 15px 0; 
+}
+.view-create .box.right .button {
+  width: 100%;
+  background: #526378;
+  border: 1px solid #29313b; 
+}
+.view-create .box.right .button:hover {
+  background: #3e4a59; 
+}
+.view-create .box.left {
+  background-color: white;
+  padding: 15px; 
+}
+.view-create .box.left h1, .view-create .box.left h2, .view-create .box.left h3, .view-create .box.left h4, .view-create .box.left h5, .view-create .box.left h6 {
+  color: #810814;
+  margin: 0 0 15px 0; 
+}
```

Comment:
Same, don't change the alignment (only for consistency, new projects should have a "normal" CSS indentation style)